### PR TITLE
Fix typos across selected folders in repo using typos-cli and ruplacer

### PR DESCRIPTION
**Pull request by cmdcolin**
This is a copy of https://github.com/monarch-initiative/mondo/pull/7684/files but only includes changes to the docs directory and mondo-edit.obo in response to the helpful review from @matentzn 

I checked out a new branch based on origin/master (this repo) and then used

 git checkout myfork/master docs
 git checkout myfork/master src/ontology/mondo-edit.obo

to just get those files from the first PR

Blocked by

- [ ] #7691
- [x] #7690 
- [x] #7692

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -109196,7 +109196,7 @@ replaced_by: MONDO:0019209
...
+def: "Kluver Bucy syndrome is a rare behavioral impairment characterized byinappropriate sexual behaviors and mouthing of objects. Other signs and symptoms, include diminished ability to visually recognize objects,loss of normal fear and anger responses, memory loss, distractibility, seizures, and dementia. It is associated with damage to the anterior temporal lobes of the brain. Cases have been reported in association with herpes encephalitis and head trauma. Treatment is symptomatic and may include the use of psychotropic medications." [https://rarediseases.info.nih.gov/diseases/6840/kluver-bucy-syndrome]
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:109199]: Comma before `include` still needed?

**Review by matentzn — APPROVED** [review]
This is absolutely wonderful @cmdcolin THANK YOU!

If you have the energy: This is a great file to run your tools on as well:
https://github.com/obophenotype/human-phenotype-ontology/blob/master/src/ontology/hp-edit.owl

Would you be willing to create a small howto guide for our academy https://oboacademy.github.io/obook/ to explain to people how to do this?

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (cmdcolin) [reply src/ontology/mondo-edit.obo:109199]: removed comma!

**Comment by cmdcolin** [comment]
it is great to hear the positive response :) i can definitely see myself running it on other ontologies now, I can look at https://github.com/obophenotype/human-phenotype-ontology/blob/master/src/ontology/hp-edit.owl

I would be happy to try to write a small guide for oboacademy too

**Comment by matentzn** [comment]
Awesome! Thank you!

**Review by sabrinatoro — APPROVED** [review]
Thank you so very much @cmdcolin !
We appreciate you taking the time to fix typos and create this PR.

**Pull request merged**